crates/libcontainer/src/rootfs/rootfs.rs

```diff
@@ -52,20 +52,34 @@ impl RootFS {
         }
 
         self.syscall
-            .mount(None, Path::new("/"), None, flags, None)?;
+            .mount(None, Path::new("/"), None, flags, None)
+            .map_err(|err| {
+                tracing::error!(
+                    ?err,
+                    ?flags,
+                    "failed to change the mount propagation type of the root"
+                );
+
+                err
+            })?;
 
         let mounter = Mount::new();
 
         mounter.make_parent_mount_private(rootfs)?;
 
         tracing::debug!("mount root fs {:?}", rootfs);
-        self.syscall.mount(
-            Some(rootfs),
-            rootfs,
-            None,
-            MsFlags::MS_BIND | MsFlags::MS_REC,
-            None,
-        )?;
+        self.syscall
+            .mount(
+                Some(rootfs),
+                rootfs,
+                None,
+                MsFlags::MS_BIND | MsFlags::MS_REC,
+                None,
+            )
+            .map_err(|err| {
```

Comment:
For example, this call to `syscall::mount` is inside the `rootfs` module. Here, mount call failure indicates an actual failure, so we log the error here.